src/helpers/tests/overTimeComparisonHelper.spec.ts

```diff
@@ -12,14 +12,14 @@ function findVisualizationObjectFixture(metaTitle: string): IVisualizationObject
     return visualizationObject.visualizationObject.content;
 }
 
-describe('popHelper', () => {
-    describe('fillPoPTitlesAndAliases', () => {
+describe('overTimeComparisonHelper', () => {
+    describe('fillDerivedMeasuresTitlesAndAliases', () => {
         it('should set title of derived measures based on master title when master is NOT renamed', () => {
             const suffixFactory = new DerivedMeasureTitleSuffixFactory('en-US');
             suffixFactory.getSuffix = jest.fn(() => ' - testing pop title');
 
             const visualizationObjectContent = findVisualizationObjectFixture('PoP');
```

Comment:
pop

Reply:
its looking for object with meta.title = 'PoP' it could be something more meaningful